Install/install.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash -e
+
+GIT_REPO=https://github.com/TomWhitwell/SlowMovie
+GIT_BRANCH=master
+PYTHON_VERSION=3
+SKIP_DEPS=false
+
+function install_linux_packages(){
+  sudo apt-get update
+
+  sudo apt-get install -y ffmpeg git python3-pip
+}
+
+function install_python_packages(){
+  # set the python3 as the default version
+  sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+  sudo update-alternatives --set python /usr/bin/python${PYTHON_VERSION}
+
+
+  sudo pip3 install setuptools -U
+  sudo pip3 install -r $LOCAL_DIR/Install/requirements.txt
+}
+
+function build_python_libraries(){
+  WAVESHARE_DIR=/home/pi/e-Paper
+
+  if [ -d "${WAVESHARE_DIR}" ]; then
+    echo -e "Updating Waveshare Drivers"
+    cd $WAVESHARE_DIR
+    git fetch
+    git pull
+  else
+    echo -e "Installing Waveshare Drivers"
+    git clone https://github.com/waveshare/e-Paper
+  fi
+
+  echo -e "$(tput setaf 1)Be patient - this takes time$(tput sgr 0)"
```

Comment:
No need to use tput. Colored text can be accomplished with [control codes](https://techstop.github.io/bash-script-colors/).
```
echo -e "\e[0;31mBe patient - this takes time\e[0m"
```

Reply:
I've used those before but in future looks always was irritated it wasn't easier to know what code = what color. I like the example in that link though, just set some variables for the colors.